Makefile

```diff
@@ -1,6 +1,7 @@
 BUILD_DIR=$(shell pwd)/build
 CODE_DIR=$(shell pwd)/iso
-VERSION=1.0.0
+VERSIONFILE=VERSION
```

Comment:
@praveenkumar IMO keeping the Version in a separate file a better way of dealing with it e.g. 1. We do not need to change the Makefile or code for every version change.
2. It is easier to find the version of the project.
3. It is easier to check the history of the file to see when the versions have been changed.